LICENSES/LicenseRef-MIT-TQ.txt

```diff
@@ -1,19 +0,0 @@
-MIT License Copyright (c) 2020 Tocqueville Group
```

Comment:
In some PRs, e.g. [here](https://github.com/serokell/tezos-packaging/pull/406), the old LicenseRef file wasn't removed, while here it was removed. Shouldn't it be consistent?

Reply:
Oh, right, because in this project this license is no longer referenced, while in the other it still might be referenced

Reply:
That's correct. Alternatively we could have changed the content of this file, but I didn't do so because it has `TQ` in the actual file name.